helpers/reminderHelper.js

```diff
@@ -0,0 +1,26 @@
+const { getGuild, saveGuild } = require('./guildData')
+const { discordTime } = require('./DateTimeHelper')
+const { foemp } = require('./foemp')
+
+async function setReminder (interaction) {
+  const guild = await getGuild(interaction.guildId)
+  const currentTime = discordTime().toEpochSecond().toString()
+  const reminderMinutes = interaction.fields.getTextInputValue('reminderTime')
+  if (isNaN(reminderMinutes)) {
+    return await interaction.reply({ content: `Je hebt het aantal minuten ingesteld (${reminderMinutes}) in een formaat dat ik niet begrijp. Probeer een integer (bv. 10), ${foemp()}!`, ephemeral: true })
+  }
+  const timeToBeReminded = discordTime().plusMinutes(interaction.fields.getTextInputValue('reminderTime')).toEpochSecond()
+  const message = interaction.fields.getTextInputValue('reminderMessage')
+  const memberId = interaction.member.id
+  guild.reminders[currentTime] = { memberId, timeToBeReminded, message }
```

Comment:
Is there any reason you still need the current time later?
If not, can't we simply use the reminderTime as key?

`guild.reminders[timeToBeReminded] = { memberId, message }`

On the same subject, I would prefer reminderTimestamp over timeToBeReminded. but yet again, this might be personal preference and is purely a suggestion